averbis/core/_rest_client.py

```diff
@@ -2002,6 +2008,22 @@ def search(self, query: str = "", **kwargs) -> dict:
         # noinspection PyProtectedMember
         return self.client._select(self.name, query, **kwargs)
 
+    @experimental_api
+    def neural_search(self, neural_search_parameter: dict, **kwargs) -> dict:
```

Comment:
IMHO parameters which are defined in the Swagger APi should also be explicitly listed here and documented - and not be part of the opaque kwargs.

Also, if there are potentially other parameters not declared by the Swagger API (e.g. some default Solr parameters), there should be an information here that these are valid and best a link to their documentation.